docs/development/testing_application_tests.rst

```diff
@@ -0,0 +1,63 @@
+.. _app_tests:
+
+Testing: Application Tests
+==========================
+
+The application test suite uses:
+
+  * Pytest_
+  * Selenium_
+  * Coveralls_
+
+The functional tests use an outdated version of Firefox chosen specifically
+for compatibility with Selenium, and a rough approximation of the most recent
+Tor Browser.
+
+.. note:: We're working on running the Selenium tests in Tor Browser.
+          See `GitHub #1629`_ for more info.
+
+.. _`GitHub #1629`: https://github.com/freedomofpress/securedrop/pull/1629
+
+.. todo:: Flesh out explanation of the application tests. Decide what's
+          important for devs to know; at a high-level, organization could be:
+
+            * Installation
+            * Running the application tests
+            * any relevant info on editing them
+
+          which matches the layout used for the Config tests.
+
+.. _Pytest: https://docs.pytest.org/en/latest/
+.. _Selenium: http://docs.seleniumhq.org/docs/
+.. _Coveralls: https://github.com/coveralls-clients/coveralls-python
+
+Installation
+------------
+
+The application tests are installed automatically in the Development
+and App-Staging VMs, based on the contents of
```

Comment:
nit: 
"Development VM" -> "development VM" 
"App-Staging VMs" -> "app-staging VMs"

(just for consistency)